docs/theme-context.rst

```diff
@@ -0,0 +1,121 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the `html_context Sphinx setting`_ in
+the ``conf.py`` file. This extra context is used by the Read the Docs Sphinx Theme
+to add additional features to the built documentation
```

Comment:
Well, we use it, but the goal of this doc is to also make it usable by the user. I think we should communicate to the user why _they_ should care about this doc, not why _we_ use it. Do they really care why we use it? If so, it could easily be a section lower in the document explaining our usage, but not part of the introduction to the feature.